src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -33,4 +34,18 @@ public CompanyProfileService(Logger logger, CompanyProfileRepository companyProf
                 companyNumber, companyProfileDao));
         return companyProfileDao;
     }
+
+    /**
+     * Updated insolvency links in company profile.
+     * @param companyProfileRequest company Profile information {@link CompanyProfile}
+     */
+    public void update(final CompanyProfile companyProfileRequest) {
```

Comment:
Ticket doesn't make it clear if we're only updating the insolvency link.
But since this is what this method does perhaps a name to better reflect this - as I feel like we'll be adding more update methods in the future and it can get confusing - `updateInsolvencyLink` ?